backend/flask.sh

```diff
@@ -18,4 +18,7 @@ fi
 export FLASK_ENV=development
 export FLASK_APP=mora.app:app
 
+# change default setting during test
+[ "$1" = "test" ] && export OS2MO_DUMMY_MODE=true
```

Comment:
This is not an ideal place to do this. It overrides the default setting but _only_ when run through `flask.sh`. Consider instead overriding it in the test class in `tests/util.py`